python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/router-adaptive-virtual-topology.j2

```diff
@@ -36,10 +36,17 @@ VXLAN gateway: Enabled
 
 #### AVT Profiles
 
-| Profile name | Load balance policy | Internet exit policy |
-| ------------ | ------------------- | -------------------- |
+| Profile name | Load balance policy | Internet exit policy | Metric Order | Jitter Threshold | Latency Threshold | Load Percentage | Loss-rate Percentage |
+| ------------ | ------------------- | -------------------- | ------------ | ---------------- | ----------------- | --------------- | -------------------- |
 {%         for profile in router_adaptive_virtual_topology.profiles | arista.avd.natural_sort('name') %}
-| {{ profile.name }} | {{ profile.load_balance_policy | arista.avd.default("-") }} | {{ profile.internet_exit_policy | arista.avd.default("-") }} |
+{%             set load_balance_policy =  profile.load_balance_policy | arista.avd.default("-") %}
+{%             set internet_exit_policy =  profile.internet_exit_policy | arista.avd.default("-") %}
+{%             set metric_order =  profile.metric_order | arista.avd.default("-") %}
+{%             set jitter =  profile.outlier_elimination.threshold.jitter | arista.avd.default("-") %}
+{%             set latency =  profile.outlier_elimination.threshold.latency | arista.avd.default("-") %}
+{%             set load =  profile.outlier_elimination.threshold.load | arista.avd.default("-") %}
+{%             set loss_rate =  profile.outlier_elimination.threshold.loss_rate | arista.avd.default("-") %}
```

Comment:
```suggestion
{%             set jitter =  profile.outlier_elimination.threshold.jitter milliseconds | arista.avd.default("-") %}
{%             set latency =  profile.outlier_elimination.threshold.latency milliseconds | arista.avd.default("-") %}
{%             set load =  profile.outlier_elimination.threshold.load percent | arista.avd.default("-") %}
{%             set loss_rate =  profile.outlier_elimination.threshold.loss_rate percent | arista.avd.default("-") %}
```

Reply:
this will require to check profile.outlier_elimination.threshold.latency is define first, if we have to add the unit.

Reply:
would use a percentage sign instead of `percent` if we go in that direction

Reply:
Current table column heading LGTM - 

`| Profile name | Load balance policy | Internet exit policy | Metric Order | Jitter Threshold (ms) | Latency Threshold (ms) | Load (%) | Loss Rate (%) |`